### PR TITLE
New pages: HTMLxElement.cite

### PR DESCRIPTION
**Pull request by estelle**
[Missing Baseline](https://github.com/openwebdocs/project/issues/214) docs

### CODE REVIEW

File: files/en-us/web/api/htmlmodelement/cite/index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+title: "HTMLModElement: cite property"
+short-title: cite
+slug: Web/API/HTMLModElement/cite
+page-type: web-api-instance-property
+browser-compat: api.HTMLModElement.cite
+---
+
+{{ApiRef("HTML DOM")}}
+
+The **`cite`** property of the {{domxref("HTMLModElement")}} interface indicates the URL of the resource explaining the modification. It reflects the {{HTMLElement("del")}} element and {{HTMLElement("ins")}} elements' [`cite`](/en-US/docs/Web/HTML/Element/ins#cite) attribute.
```
Comment (wbamberg) [inline files/en-us/web/api/htmlmodelement/cite/index.md:11]:
```suggestion
The **`cite`** property of the {{domxref("HTMLModElement")}} interface indicates the URL of the resource explaining the modification. It reflects the `cite` attribute of the  {{HTMLElement("del")}} element and {{HTMLElement("ins")}} elements.
```

Just slightly clearer sentence structure, and IMO better not to link to just one of them.

File: files/en-us/web/api/htmlmodelement/cite/index.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Value
+
+A string; a URL.
```
Comment (wbamberg) [inline files/en-us/web/api/htmlmodelement/cite/index.md:15]:
```suggestion
A string representing a URL.
```

File: files/en-us/web/api/htmlquoteelement/cite/index.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+browser-compat: api.HTMLQuoteElement.cite
+---
+
+{{ApiRef("HTML DOM")}}
+
+The **`cite`** property of the {{domxref("HTMLQuoteElement")}} interface indicates the URL for the source of the quotation.. It reflects the {{HTMLElement("q")}} element's [`cite`](/en-US/docs/Web/HTML/Element/q#cite) attribute.
+
+## Value
+
+A string; a URL.
```
Comment (wbamberg) [inline files/en-us/web/api/htmlquoteelement/cite/index.md:15]:
```suggestion
A string representing a URL.
```

File: files/en-us/web/api/htmlquoteelement/cite/index.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+## Example
+
+```js
+const quote = document.querySelector("q");
+console.log(`Orignal source: ${quote.cite}`); // the current value
```
Comment (wbamberg) [inline files/en-us/web/api/htmlquoteelement/cite/index.md:21]:
```suggestion
console.log(`Original source: ${quote.cite}`); // the current value
```

**Comment by estelle** [comment]
Thanks @wbamberg!

**Review by wbamberg — APPROVED** [review]
👍 thank you!

**Pull request merged**